dashboard/models.py

```diff
@@ -280,9 +282,8 @@ def get_course_resource_type(self, course_id):
 class Resource(models.Model):
     id = models.AutoField(primary_key=True, verbose_name="Table Id")
     resource_type = models.CharField(max_length=255, verbose_name="Resource Type")
-    resource_id = models.CharField(blank=True, db_index=True, max_length=255, null=False, verbose_name="Resource Id")
+    resource_id = models.CharField(unique=True, blank=False, db_index=True, max_length=255, null=False, verbose_name="Resource Id")
```

Comment:
`blank=False` and `null=False` maybe unnecessary here; those are defaults.
https://docs.djangoproject.com/en/3.0/ref/models/fields/#null

Reply:
Okay, I didn't notice what the defaults were I can remove these.